motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -159,7 +159,8 @@ typedef enum
 	ROS_RESULT_NOT_READY_NOT_STARTED,
 	ROS_RESULT_NOT_READY_WAITING_ROS,
 	ROS_RESULT_NOT_READY_SKILLSEND,
-	ROS_RESULT_NOT_READY_PFL_ACTIVE
+	ROS_RESULT_NOT_READY_PFL_ACTIVE,
+	ROS_RESULT_NOT_READY_INC_MOVE_ERROR

```

Comment:
We need to update the `NotReadyCode` enum:

https://github.com/ros-industrial/motoman/blob/9ade90acfe5835ca38fddcf6ac9cf589a452f449/motoman_driver/include/motoman_driver/simple_message/motoman_motion_reply.h#L100-L116

Reply:
We'll do that in a separate PR.

Reply:
See #438.